client/bootstrap_demo_community.py

```diff
@@ -320,6 +320,53 @@ def test_faucet(client, cid):
         exit(1)
     print('Faucet closed', flush=True)
 
+
+def test_democracy(client, cid):
+    print('Starting democracy test...')
+    client.next_phase()
+    client.next_phase()
+    client.next_phase()
+    #phase is 9, registering
+    print(client.purge_community_ceremony(cid, 1, 8))
+    register_participants_and_perform_meetup(client, cid, accounts)
+    cindex = 9
+
+    # registering of cindex 10
+    client.next_phase()
+
+    claim_rewards(client, cid, "//Alice", pay_fees_in_cc=False)
+    client.await_block(1)
+
+    client.next_phase()
+    client.next_phase()
+    client.next_phase()
+    # cindex is now 11
+
+
+    client.await_block(1)
+    client.submit_set_inactivity_timeout_proposal("//Alice", 8)
+    client.await_block(1)
+    proposals = client.list_proposals()
+    print(proposals)
+    if('id: 1' not in proposals):
+        print(f"Proposal Submission failed")
+        exit(1)
+
+    print('proposal submitted')
+    # vote with all reputations gathered so far
+    client.vote("//Alice", 1, "aye", [[cid, cindex]])
+    client.vote("//Bob", 1, "aye", [[cid, cindex]])
+    client.vote("//Charlie", 1, "aye", [[cid, cindex]])
+
+
+    client.await_block(21)
+    client.update_proposal_state("//Alice", 1)
+    proposals = client.list_proposals()
+    print(proposals)
+    if('Approved' not in proposals):
```

Comment:
within the same testing time we could test 3 proposals to increase test coverage:
1. 2x Aye, 1x Nay
2. 1x Aye, 2x Nay
3. no votes

Moreover, it would be nice to test enactment too, not just approval
(pardon, I know you haven't asked for the review just yet ;-)

Reply:
sure! now in the light of very limited time remaining from my side, where to put the focus? this, or the remaining tasks for the accounting dashboard?

Reply:
updated tracking issue accordningly
https://github.com/encointer/pallets/issues/347